Sources/StreamChat/Utils/UniqueId.swift

```diff
@@ -6,5 +6,5 @@ import Foundation
 
 extension String {
     /// Creates and returns a new unique id every time the variable is accessed.
-    static var newUniqueId: String { UUID().uuidString }
+    static var newUniqueId: String { ULID().ulidString }
```

Comment:
There are many other places where we are using UUID. Should we move those too?

Reply:
Nope :)

Reply:
The internal workers, runners, queues, test helpers wouldn't really leverage this format.